habitat-lab/habitat/core/registry.py

```diff
@@ -181,23 +181,6 @@ def register_dataset(cls, to_register=None, *, name: Optional[str] = None):
             "dataset", to_register, name, assert_type=Dataset
         )
 
-    @classmethod
-    def register_action_space_configuration(
```

Comment:
We deprecate registering new action spaces for sim. If you want to add new actions, you can directly add them to the action space of lab. YOU CAN NO LONGER create new sim actions. The only sim actions we have will be hardcoded. They are `move forward` `turn left` and `turn right`. We use them for pathfinding and other tools.